packages/extension-ui/src/Popup/CreateAccount/CreateAccount.test.tsx

```diff
@@ -185,4 +190,26 @@ describe('Create Account', () => {
       expect(wrapper.find('[data-button-action="add new root"] button').prop('disabled')).toBe(false);
     });
   });
+
+  describe('Account gets cached and restored', () => {
```

Comment:
+1 (and another +1 and ...) for adding UI tests. I suck at these.

Reply:
trying to get better at this, but there's still a long way.
I'm only partly satisfied by these tests because:
- if I test [here](https://github.com/polkadot-js/extension/pull/466/files#diff-1f14364b73a398aa6bfdcd7926ad6c99R211) for
```diff
- expect(getAccountCache).toBeCalled();
+ expect(getAccountCache).toBeCalledTimes(1);
```

it fails because it's called twice.. but I don't understand why.